### ISSUE TITLE
Adding slurs in normal mode de-select note(s)

### ISSUE THREAD

**Issue by mgavioli** [musescore/MuseScore#19025]
### Issue type

UI bug

### Bug description

**Context**: Version 4.1.1, Linux Mint 21.2.

**Description**:
After entering a slur in normal mode, exiting the slur editing with [ESC] leaves nothing selected.

In previous versions, after ending the slur editing, the note where the slur have been started was still selected and one could immediately go on working on the score.

Instead, the current situation requires:
1) to reach for the mouse,
2) point the first slurred note,
3) click to select it,
4) and then replace the hand(s) on the keyboard (either the computer's or the MIDI's).

This is cumbersome and breaks the workflow every few notes (as slurs are rather commons). In practice, I noticed it takes me almost twice as much to edit a score in ver. 4.1.1 than I took me on ver. 2.3.2; this means a drastic decrease in productivity.

Note that the [manual](https://musescore.org/en/handbook/4/slurs#normal-method-1) describes the previous (faster) functioning.

### Steps to reproduce

1. Open any score
2. select a note
3. press [S], [ESC]

**Result**: a slur is added but eventually no note is selected;
**Expected result**: one note (possibly the one on which the process was started) is still selected.

### MuseScore Version

4.1.1

### Regression

Yes, this used to work in MuseScore 3.x and now is broken

### Operating system

Runnig Linux Mint 21.2 here, but I expect this to happen on any OS

**Comment by bkunda** [comment]
I'm no longer sure this should be fixed (or indeed, that it is a bug). 

Having the slur selected allows you to immediately operate on it, which is useful. 

And if you are inputting notes and want to also add slurs at the same time, then Alt/Option+Arrow allows you to get back to whichever note you want to select, so I don't believe there's an accessibility issue here.

Moreover, while in MS3 the note remained selected after adding a slur, it still wasn't possible to perform a subsequent operation on it; all up/down/shift/command operations still affected the slur, not the note. This actually feels more like a bug than the current behaviour in MS4+, because the note that's actually selected doesn't get altered by any keystrokes. At least in MS4+, the keystrokes affect the object that appears selected. I wouldn't want to change this. 

_MS3 behaviour_

https://github.com/musescore/MuseScore/assets/86290556/a96faa58-b2d5-46e1-90f0-86367209aca7

**Comment by mgavioli** [comment]
I probably did not express myself clearly enough.

The problem (if you not like call it a bug) is NOT that the slur is selected after being added, this happens since a lot of MS versions and is fine, as it conveniently allows to work ON THE SLUR as soon it is added.

The problem is that deselecting the slur with ESC, nothing become selected instead (once upon a time, it used to be the note from which the slut was created) and requires some additional operations (and some mouse / keyboard switches), which takes what it is actually A LOT of time, compared with the previous workflow.

Simply the fact that it takes me roughly the double to input a score piece with an amount of sluts not unusually small, for me amount to a regression.
